assets/js/phoenix_live_view.js

```diff
@@ -182,6 +182,15 @@ let recursiveMerge = (target, source) => {
   }
 }
 
+let toArray = function(nodeList) {
+  let array = []
```

Comment:
Is there a reason not to use `Array.slice.apply` here?

```suggestion
  return [].slice.apply(nodeList)
```